### PR TITLE
Update layers.py

### PR DESCRIPTION
**Pull request by shreyneil**
I have added a check on beta with reference to the following issue raised.
https://github.com/tensorflow/tensorflow/issues/11673

Please verify and get back.
Thank you.

I will add update the unit test cases to verify this patch.

### CODE REVIEW

**Review by vrv — APPROVED** [review]

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

**Comment by vrv** [comment]
Please add the test as requested in the previous version of the CL, otherwise this looks great, thanks!

**Comment by shreyneil** [comment]
https://github.com/tensorflow/tensorflow/pull/13864 contains the updated unit test case.

File: tensorflow/contrib/layers/python/layers/layers_test.py [hunk 1]
```diff
@@ -3688,7 +3688,17 @@ def test_1d_invalid_input(self):
                                    'rank of x must be at least 2 not: 1'):
         x = constant_op.constant([[]], shape=[0])
         _layers.legacy_fully_connected(x, 2, activation_fn=nn_ops.softmax)
-
-
+  
+class zero_debias_moving_mean(test.TestCase):
```
Comment (vrv) [inline tensorflow/contrib/layers/python/layers/layers_test.py]: In this case, you can just add the test to the existing BatchNormTest suite, no need to add a new ones.

In general though, you can check the style of other class names in this file; one should use CamelCase test names here if you were adding a new class.

File: tensorflow/contrib/layers/python/layers/layers_test.py [hunk 2]
```diff
@@ -3688,7 +3688,17 @@ def test_1d_invalid_input(self):
...
+  
+  def Error_in_tf.contrib.layers.batch_norm_when(self):
```
Comment (vrv) [inline tensorflow/contrib/layers/python/layers/layers_test.py]: test names should start with 'test':

def testBatchNormCenterFalse

File: tensorflow/contrib/layers/python/layers/layers_test.py [hunk 3]
```diff
@@ -3688,7 +3688,17 @@ def test_1d_invalid_input(self):
...
+    import tensorflow as tf
```
Comment (vrv) [inline tensorflow/contrib/layers/python/layers/layers_test.py]: In internal tests, you have to import the modules individually, you cannot import tensorflow here.  So you should use something like:

a = array_ops.placeholder(dtypes.float32, ...)

Take a look at the existing batch norm tests and use that as a guide for how to write the test where center=False

**Review by vrv — CHANGES_REQUESTED** [review]

**Comment by vrv** [comment]
It looks like you may be having trouble updating this PR to include all the changes  -- perhaps it makes sense to resend a new fresh PR with all of the requested changes (layers.py and layers_test.py change in the same PR).  Thanks!
